src/stave.ts

```diff
@@ -189,9 +189,7 @@ export class Stave extends Element {
     this.end_x += shift;
     for (let i = 0; i < this.modifiers.length; i++) {
       const mod = this.modifiers[i];
-      if (mod.x !== undefined) {
-        mod.x += shift;
-      }
+      mod.setX(mod.getX() + shift);
```

Comment:
What if `mod.x` is `undefined`? Seems like we should catch that and throw an exception earlier on.

Reply:
mod.getX() returns a number, undefined is not possible.